### DISCUSSION TITLE
Install for all users? (windows)

### DISCUSSION THREAD

**Discussion by smood922** [mistweaverco/bananas#34] [Q&A] [answered]
I'm looking into administratively pre-deploying Bananas for Windows end-users with our RMM. Currently, the Windows installer appears to install per user and doesn't have a per machine/all users parameter. I've looked into electron build for NSIS enough to know that there are parameters to allow optional per machine install. Is this something you're open to including in the future?

**Comment by gorillamoe** [comment, marked as answer]
I don't see anything blocking that. Will look into it!

https://www.electron.build/nsis.html#permachine

> If `oneClick` is `false` and `perMachine` is `false` (default): install mode installer page.

The upcoming release will have this implemented.
